src/helpers.js

```diff
@@ -144,6 +154,11 @@ async function deploy() {
     assets: assets,
   };
 
+  const deployOptionsConfig =
```

Comment:
I think this part could be a little cleaner. There are three different variables for deployment options. I think we can get that down to one and get rid of the ternary too:
```
  const deployOptions = {
    env: {
      TWILIO_ACCOUNT_SID: this.twilioClient.accountSid,
      TWILIO_API_KEY_SID: this.twilioClient.username,
      TWILIO_API_KEY_SECRET: this.twilioClient.password,
      API_PASSCODE: pin,
      API_PASSCODE_EXPIRY: expiryTime,
    },
    pkgJson: {},
    functionsEnv: 'dev',
    functions: [
      {
        name: 'token',
        path: '/token',
        content: fn,
        access: 'public',
      },
    ],
    assets: assets,
  };

  if (appInfo && appInfo.sid) {
    deployOptions.serviceSid = appInfo.sid;
  } else {
    deployOptions.serviceName = APP_NAME;
  }

  try {
    await serverlessClient.deployProject(deployOptions);
    cli.action.stop();
  } catch (e) {
    console.error('Something went wrong', e);
  }
```
This snippet is also assuming that `appInfo` is passed in as a parameter. What do you think?

Reply:
Agreed 👍